package.json

```diff
@@ -11,14 +11,16 @@
   "homepage": "https://github.com/nodeca/pako",
   "contributors": [
     "Andrei Tuputcyn (https://github.com/andr83)",
-    "Vitaly Puzrin (https://github.com/puzrin)"
+    "Vitaly Puzrin (https://github.com/puzrin)",
```

Comment:
Depends. In europe particularly in civil law system you should add author except for minor contrbutions... Safer to do before distribution